sources/packages/backend/apps/queue-consumers/src/processors/assessment/start-application-assessment.processor.ts

```diff
@@ -31,6 +32,22 @@ export class StartApplicationAssessmentProcessor {
     const processSummary = new ProcessSummary();
     try {
       processSummary.info("Processing the start assessment job.");
+      const assessment = await this.studentAssessmentService.getAssessmentById(
```

Comment:
Should we refactor the method `getApplicationDynamicData` to return assessment status(and also rename method) and used it here instead of going to db twice?

![image](https://github.com/bcgov/SIMS/assets/54600590/9a759375-30c7-4147-bc06-bdf22af6b572)

Reply:
I would prefer to keep it as it is. The `getApplicationDynamicData` was created to be dedicated to get the dynamic data in a "typed" way. If needed we can change the `getAssessmentById` to also get the workflow name but either way it does not seen part of the PR.